Include/cpython/pyatomic.h

```diff
@@ -545,6 +545,61 @@ static inline Py_ssize_t
 _Py_atomic_load_ssize_acquire(const Py_ssize_t *obj);
 
 
+// --- _Py_atomic_memcpy / _Py_atomic_memmove ------------
+
+static inline void *
+_Py_atomic_memcpy_ptr_store_relaxed(void *dest, void *src, Py_ssize_t n)
+{
+    size_t size = (size_t)n;
+    assert(_Py_IS_ALIGNED(dest, sizeof(void *)));
+    assert(_Py_IS_ALIGNED(src, sizeof(void *)));
+    assert(size % sizeof(void *) == 0);
+
+    if (dest != src) {
+        void **dest_ = (void **)dest;
+        void **src_ = (void **)src;
+        void **end = dest_ + size / sizeof(void *);
+
+        for (; dest_ != end; dest_++, src_++) {
+            _Py_atomic_store_ptr_relaxed(dest_, *src_);
+        }
+    }
+
+    return dest;
+}
+
+static inline void *
+_Py_atomic_memmove_ptr_store_relaxed(void *dest, void *src, Py_ssize_t n)
```

Comment:
I think these two functions should take size_t like memcpy does

Reply:
Would you add tests for these in this PR as well? Then we can remove the suppressions for these as well in this PR.

Reply:
Can you try building with a clean checkout?

Reply:
> But the weird thing is I couldn't trigger TSAN warnings when reverted to non-atomic memcpy and memmove. Didn't try too hard though.

The tests should trigger TSAN warnings, maybe adding a different thread with non locking reads would trigger it?